web_src/css/base.css

```diff
@@ -1569,8 +1569,7 @@ a.ui.active.label:hover {
 }
 
 .blame .code-inner {
-  white-space: pre;
```

Comment:
I think this will likely break at least the leading indendation of the code. I think we absolutely need `pre` rendering.